index.js

```diff
@@ -15,8 +15,7 @@ import {CredentialsContainer} from './CredentialsContainer.js';
 import {PermissionManager} from './PermissionManager.js';
 import {WebCredential} from './WebCredential.js';
 
-const DEFAULT_MEDIATOR = 'https://authn.io/mediator' + '?origin=' +
-  encodeURIComponent(window.location.origin);
```

Comment:
Is that origin param no longer needed?  Doesn't look like it's added back in this patch?

Reply:
Yeah, it's no longer needed. This patch is, in part, to help remove that.